pkg/ddevapp/composer.go

```diff
@@ -17,23 +16,13 @@ func (app *DdevApp) Composer(args []string) (string, string, error) {
 		return "", "", fmt.Errorf("failed to process pre-composer hooks: %v", err)
 	}
 
-	// Prevent Composer from debugging when Xdebug is enabled
-	env := []string{"XDEBUG_MODE=off"}
-	// Let Composer know which binary to run from the PATH
-	path, _, err := app.Exec(&ExecOpts{
-		Cmd: "echo $PATH",
-	})
-	path = strings.Trim(path, "\n")
-	if err == nil && path != "" {
-		env = append(env, "PATH="+path)
-	}
-
 	stdout, stderr, err := app.Exec(&ExecOpts{
 		Service: "web",
 		Dir:     app.GetComposerRoot(true, true),
 		RawCmd:  append([]string{"composer"}, args...),
 		Tty:     isatty.IsTerminal(os.Stdin.Fd()),
-		Env:     env,
+		// Prevent Composer from debugging when Xdebug is enabled
+		Env: []string{"XDEBUG_MODE=off"},
```

Comment:
Is this related to the revert here? Or just a good idea? I thought it was in here before.